code/hsec-tools/hsec-tools.cabal

```
@@ -32,8 +32,11 @@ library
                       , Security.Advisories.HsecId
                       , Security.Advisories.Parse
                       , Security.Advisories.Convert.OSV
+                      , Security.Advisories.Generate.HTML
                       , Security.OSV
     build-depends:    base >=4.14 && < 4.19,
+                      directory ^>= 1.3.6.1,
```

Directory is already capped by process, so why not using a simpler ?
```suggestion
                      directory < 2,
```